test/framework/common_constants.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
```

Comment:
Our codestyle prefer ifdefs guards

Reply:
Ok, but is it better in some way? I do not see any advantages of `\#define` guards over the `\#pragma` ones. Maybe we should move to using `\#pragma one` (of course in another commit)?

Reply:
afaik pragma once is considered deprecated in some versions of our target compilers

Reply:
Is there a list of our target compilers so that I could check it?

Reply:
IBM for zOS